app/templates/index.html

```
@@ -2,33 +2,96 @@
+            <path d="M11.241 9.817c-.36.275-.801.425-1.255.427-.428 0-.845-.138-1.187-.395L0 2.6V14a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V2.5l-8.759 7.317Z"/>
+          {% endif %}
+          </svg>
+        </span>
+        <input
+          class="rounded-none rounded-e-lg border border-gray-300 block flex-1 min-w-0 w-full px-3 py-2"
+          type="{{ type }}"
+          {% if autocomplete %}autocomplete="{{ autocomplete }}"{% endif %}
+          id="{{ id }}"
+          name="{{ name }}"
+          {% if required %}aria-required="true" required{% endif %}
+        >
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)